src/test/acceptance/rest/test_rest_analyze_firmware.py

```diff
@@ -40,6 +40,7 @@ def _rest_upload_firmware(self):
             'binary': standard_b64encode(file_content).decode(),
             'file_name': 'test.zip',
             'device_name': 'test_device',
+            'device_part': 'full',
```

Comment:
care to check for this in result?

Reply:
tested in test/acceptance/rest/test_rest_analyze_firmware.py
+ BugFix in helperFunctions/object_conversion.py